### PR TITLE
Revert "Adds ERP"

### PR DESCRIPTION
**Pull request by SomeguyManperson**
Reverts #8088

### CODE REVIEW

**Comment by Codeatmos** [comment]
merge

**Comment by SomeguyManperson** [comment]
tested over 1000 times

File: code/modules/jobs/job_types/assistant.dm [hunk 1]
```diff
@@ -37,3 +37,7 @@ Assistant
 		uniform = /obj/item/clothing/under/rank/clown
 	shoes = /obj/item/clothing/shoes/clown_shoes
 	mask = /obj/item/clothing/mask/gas/clown_hat
+
+/datum/outfit/job/assistant/post_equip(mob/living/carbon/human/H)
```
Comment (Hopekz) [inline code/modules/jobs/job_types/assistant.dm:41]: Moths

**Review by Hopekz — APPROVED** [review]

File: code/modules/jobs/job_types/assistant.dm (hunk 1 above)
Comment (SomeguyManperson) [reply code/modules/jobs/job_types/assistant.dm:41]: marke as ic

Comment (Hopekz) [reply code/modules/jobs/job_types/assistant.dm:41]: MOOOOOTHS

**Pull request closed without merging**